src/components/drag-and-drop/custom-drag-layer/custom-drag-layer.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DragLayer } from 'react-dnd';
+import classNames from 'classnames';
+
+const collect = (monitor) => {
+  const item = monitor.getItem();
+  return {
+    currentOffset: monitor.getSourceClientOffset(),
+    item,
+    draggableNode: item ? item.draggableNode : null,
```

Comment:
should this be listed as a prop type in `WithDrag` so it's clear where you set it?